catalyst_voices/lib/widgets/text_field/voices_text_field.dart

```diff
@@ -131,6 +133,7 @@ class _VoicesTextFieldState extends State<VoicesTextField> {
     final textTheme = theme.textTheme;
 
     final labelText = widget.decoration?.labelText ?? '';
+    final resizable = widget.maxLines == null && widget.minLines == null;
```

Comment:
Does it make sense to have it guarded with parameter? I'm thinking about mobile where its not natural to have this resize. 

Maybe nullable `resizableEnabled` where by default it's checked against `CatalystPlatform.isDesktop`?

Reply:
Ok, I'll add explicit param for this. TextField doesn't allow to be resizable if minLines or maxLines is specified so we need to make sure they are null.

Reply:
I made this param true by default is we're on CatalystPlatform.isDesktop or CatalystPlatform.isWebDesktop